peering_manager/jinja2/filters.py

```diff
@@ -808,6 +808,16 @@ def indent(value, n, chars=" ", reset=False):
 
     return r
 
+def bfds(value):
+    """
+    Returns all the BFDs that have at least one session configured on the router.
+    """
+    if not isinstance(value, Router):
+        raise ValueError("value is not a router")
+    r = BFD.objects.filter(
+        Q(directpeeringsession__router=value) | Q(internetexchangepeeringsession__ixp_connection__router=value)
+    ).distinct()
```

Comment:
I think this particular bit can go into the `Router` class as a getter kind of function such as `get_bfd_configs` or something like that. You would the n just call this function in this filter.